### DISCUSSION TITLE
How to change the default behavior of hyperlinks?

### DISCUSSION THREAD

**Discussion by windmgc** [wezterm/wezterm#2289] [Q&A] [answered]
By default, the hyperlinks will be highlighted with underscores and can be directly clicked to open the browser. How to change its default behaviour? (like I want it not to be directly opened, to make it convenient to be selected with a double-click. I can define some additional keys combined with a mouse click to get it opened)

**Comment by MuhammedZakir** [comment, marked as answer]
Example: https://wezfurlong.org/wezterm/config/mouse.html?highlight=hyperlink#configuring-mouse-assignments
Important note: https://wezfurlong.org/wezterm/config/mouse.html?highlight=hyperlink#gotcha-on-binding-an-up-event-only

**Reply by windmgc** [reply]
Thank you so much!
